repository/environment_repo.go

```diff
@@ -0,0 +1,43 @@
+package repository
+
+import (
+	db "osvauld/db/sqlc"
+	dto "osvauld/dtos"
+	"osvauld/infra/database"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+)
+
+func AddEnvironment(ctx *gin.Context, args dto.AddEnvironment, caller uuid.UUID) (uuid.UUID, error) {
```

Comment:
we should keep the code inside a repo layer function to a single line. This will make removal of repo layer in the near future easier

Reply:
did you mean to change args type from dto.AddEnvironment to db.AddEnvironmentParams?